src/main/java/core/basesyntax/figures/FigureArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.figures;
+
+public interface FigureArea {

```

Comment:
```suggestion
public interface AreaCalculator {

```

as we can use this interface not only for Figures, but for some flats or fields